WordPress/Classes/ViewRelated/Blog/CreateNewBlog/CreateNewBlogViewController.m

```diff
@@ -12,12 +12,12 @@
 #import "BlogService.h"
 #import "ContextManager.h"
 #import "Constants.h"
-#import <WordPressShared/NSString+XMLExtensions.h>
-#import <WordPressShared/WPNUXUtility.h>
-#import <WordPressShared/WPStyleGuide.h>
+#import <WordPressShared/WordPressShared.h>
+#import <WordPressUI/WordPressUI.h>
+#import <WordPressKit/WordPressKit.h>
 #import "WordPress-Swift.h"
-@import WordPressKit;
-@import WordPressShared;
```

Comment:
is there any reason to prefer `#import <>` for our own frameworks, instead of `@import` or is it just a style thing?

Reply:
`@import` is called ["Semantic Import"](https://stackoverflow.com/questions/18947516/import-vs-import-ios-7). It's slightly different (you don't need to add the target framework in the project settings)... but in these cases, i honestly do that just because it's shorter:

```
#import <WordPressKit/WordPressKit.h>
@import WordPressKit
```

🔥